db/migrate/20230327111836_create_likes.rb

```diff
@@ -0,0 +1,9 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :author, null: false, foreign_key: { to_table: :users}
+
```

Comment:
- [ ] Please add a reference to address the relationship between the likes table and the posts table.

![image](https://user-images.githubusercontent.com/68796232/228015883-6b460608-dd2c-4942-9b47-4ab6289154db.png)

[Hint] it will be like that:
```ruby
t.references :post, null: false, foreign_key: true
```